### DISCUSSION TITLE
Packaged version of the app

### DISCUSSION THREAD

**Discussion by reshBC1779** [mayooear/ai-pdf-chatbot-langchain#368] [Q&A]
Hi all, this looks neat. I was wondering if there is a way (once I set it up correctly that is), to package it up as an app or .exe program that someone can click on have the localhost be up and running on their machine? I feel like it's farfetched but open to see if anyone has any thoughts on this. 

Ideally, as an internal tool for people to do PDF reviews quick mostly for large building design code analysis. I can provide more context if this isn't clear. Please let me know. Thank you.

**Comment by snowsecure** [comment]
Check out nativefier as a potential solution.
